Library/Formula/gtk+3.rb

```diff
@@ -22,9 +25,26 @@ class Gtkx3 < Formula
   depends_on "pango"
   depends_on "glib"
 
+  if build.with?("icons")
+    # Adwaita icon theme dependencies
+    depends_on "gettext" => :build
+    depends_on "icon-naming-utils" => :build
+    depends_on "intltool" => :build
+    depends_on "librsvg"
```

Comment:
good catch

I had a look at the librsvg source code and apparently if gtk+3 is found at configure time then a simple svg viewer called `rsvg-view-3` will get built and installed.

what's the best course of action here?